From 3434b9f8a61d14003d4039607efede6c78ffb383 Mon Sep 17 00:00:00 2001
From: Dmitry Ornatsky <cdo021c@motorola.com>
Date: Tue, 3 Nov 2015 11:16:55 -0600
Subject: [PATCH 440/959] IKKRNBSP-3486 arm/dt: 8952: disable cirrus IC

Cirrus IC panics on latest SW. Disable it to let
other ppl work.

Change-Id: Iba1109c9f963e3a7954bfbba3c6538ae19eeba0c
Signed-off-by: Dmitry Ornatsky <cdo021c@motorola.com>
Reviewed-on: http://gerrit.mot.com/796972
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Satya Prasad Kottisa <fvgm76@motorola.com>
Reviewed-by: Christopher Fries <cfries@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
index a2eb0f11d39..2472aa45cf2 100644
--- a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
@@ -767,6 +767,7 @@
 
 &i2c_2 { /* BLSP1 QUP2 */
 	cs35l34@40 {
+		status = "disabled";
 		compatible = "cirrus,cs35l34";
 		reg = <0x40>;
 		cirrus,reset = <&msm_gpio 94 0>;
-- 
2.11.0

